microsoft-edge/progressive-web-apps-chromium/whats-new/PWAs.md

```diff
@@ -0,0 +1,115 @@
+
+## Protocol Handlers Origin Trial 
+
+You can now register your PWA to handle specific protocols with the host operating system. The Windows trial for this is now available. You can register your origin for the trial at the [origin trial signup page][MicrosoftDeveloperMicrosoftEdgeOriginTrialsWebAppProtocolHandlerRegistrationRegistration].
+
+Learn more about using protocol handlers with your PWA at our [experimental features page][ExpProtocolHandlers].
+
+## Streamlined App Info Menus
+The app info menu is what is shown when a user clicks the 3 dot settings menu in the apps title bar. We've updated this menu and streamlined the experience for the user by:
+* bringing the app publisher information to the top level and making it the first thing a user sees
+* moving the privacy information and controls into a dedicated 2nd level "Privacy" menu
+* moving content related tools into a dedicated 2nd level "More tools" menu
+
+This change brings the settings menu more in line with what users expect from a desktop app experience and less like browser UI. 
+
+#### Updated App Info Menu
+:::image type="complex" source="media/app-info.png" alt-text="Streamlined app info menu" lightbox="media/app-info.png":::
+   Screen shot of the new streamlined app info menu
+:::image-end:::
+
+#### Dedicated Privacy Menu
+:::image type="complex" source="media/privacy-menu.png" alt-text="Privacy controls found in the dedicated Privacy menu" lightbox="media/privacy-menu.png":::
+   Privacy controls found in the dedicated Privacy menu
+:::image-end:::
+
+#### Dedicated More Tools Menu
+:::image type="complex" source="media/more-tools.png" alt-text="Content related tools menu" lightbox="media/more-tools.png":::
+   Content related tools now found in More tools menu
+:::image-end:::
+
+## Post Install Flyout
+After a PWA is installed from the Edge browser on Windows, users will now be able to select from four options to easily launch their apps: 
+* Pin to Start 
+* Pin to taskbar 
+* Create Desktop shortcut
+* Auto-start on device login.
+
+For convenience, this flyout is shown the first time the app is launched.
+
+This feature is being rolled out gradually to all users. In the meantime, if you'd like to use this feature, navigate to `edge://flags` and enable the flag *Web Apps Post Install Dialog*.
+
+#### Post install Flyout example
+:::image type="complex" source="media/postInstallFlyout.png" alt-text="Users can select to pin, create shortcut or run on device login" lightbox="media/postInstallFlyout.png":::
+   Screen shot of the post-install flyout dialog box
+:::image-end:::
+
+## Restore Web Apps
+Installed sites and PWAs that were running prior to an unexpected shutdown will now restore (be restarted) when the system recovers. 
+
+This can occur due to process failure, system restart, power outage etc. Prior to this change, installed sites and PWAs had indeterminant behavior upon system restore.  
```

Comment:
```suggestion
This can occur due to process failure, system restart, power outage etc. Prior to this change, installed sites and PWAs had indeterminant behavior upon system restore.  
```
Style guide re: "etc": https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/etc